### PR TITLE
Wrap config type into customize libvirt config

### PR DESCRIPTION
**Pull request by chunfuwen**
test result is under preparing.

### CODE REVIEW

**Comment by chunfuwen** [comment]
1)https://github.com/autotest/tp-libvirt/pull/3023
2)https://github.com/autotest/tp-libvirt/pull/3012

This change can help avoid  some duplicated code calling similar to above PRs

**Comment by chunfuwen** [comment]
test result is under preparing.

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -3575,7 +3576,11 @@ def customize_libvirt_config(params,
     :param config_object: an existing utils_config.LibvirtConfigCommon object
     :return: utils_config.LibvirtConfigCommon object
     """
-
+    # Hardcode config_type to virtqemud under modularity daemon mode when config_type="libvirtd"
+    # Otherwise accept config_type as it is
+    if utils_split_daemons.is_modular_daemon():
+        if config_type in ["libvirtd"]:
+            config_type = "virtqemud"
```
Comment (Yingshun) [inline virttest/utils_test/libvirt.py:3583]: The function "customize_libvirt_config" was used to modifying config keys for 'libvirtd', 'qemu', 'sysconfig' and so on, would you please create a new function to get the config_type?

The configuration key of libvirtd does not map directly to those in virtqemud. And some virt*d daemons have the same configuration key, but for different functions. So I'm not sure if it works... @yafu-1  Could you please help review it?

Comment (chunfuwen) [reply virttest/utils_test/libvirt.py:3583]: if carefully check the difference between this PR and https://github.com/autotest/tp-libvirt/pull/3023/files or https://github.com/autotest/tp-libvirt/pull/3012, the only difference is moved below logic from outside of customize_libvirt_config to inside of customize_libvirt_config :
config_type = 'libvirtd'
if utils_split_daemons.is_modular_daemon():
            config_type = "virtqemud"

Comment (yafu-1) [reply virttest/utils_test/libvirt.py:3583]: I think the pr works well. The default config file is 'virtqemud.conf' if not specified and can set to different split daemon conf file if needed.

**Review by yafu-1 — APPROVED** [review]

**Review by kylazhang — APPROVED** [review]

**Pull request merged**